### PR TITLE
Added 8px margin between buttons in CreateGameDialog

### LINKED ISSUE #614: [Feature]: Add 4px margin between buttons in `CreateGameDialog`
### Feature Summary

The two buttons in the `CreateGameDialog` are too close together. We should add margin between them

### Detailed Description

The `CreateGameDialog` used in the `HomeView` has a button to cancel and a button to create the game/submit the form. They should be put at least 4px apart. To do this we should use the existing [vuetify spacing classes](https://vuetifyjs.com/en/styles/spacing/) to apply margin right to the cancel button with `class="mr-1"`. 

It would be good to try out 8px with `class="mr-2"` for comparison and pick whichever looks better.

See the current dialog to notice how the buttons touch:
image

### PR DESCRIPTION
**Pull request by sinhashlok**
## Issue number #614 

Relevant #614 
- Resolves #614 

## Please check the following

- [x] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [x] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
I don't think additional details are required as it's just adding mr-2 (margin right: 8px)

### CODE REVIEW

File: src/components/CreateGameDialog.vue [hunk 1]
```diff
@@ -53,6 +53,7 @@
     <template #actions>
       <v-form>
         <v-btn
+          class="mr-2"
```
Comment (seriouslysean) [inline src/components/CreateGameDialog.vue:56]: @itsalaidbacklife this looks ok to me, pretty simple. You're more familiar with all the CSS stuff.

**Review by seriouslysean — COMMENTED** [review]
Leaving finally approval to @itsalaidbacklife

File: src/components/CreateGameDialog.vue (hunk 1 above)
Comment (sinhashlok) [reply src/components/CreateGameDialog.vue:56]: Thanks.
Yeah, it's more similar to Tailwind.

Comment (itsalaidbacklife) [reply src/components/CreateGameDialog.vue:56]: Yes this is right and yes it's a similar approach to tailwind. Vuetify bakes some of these styles in out of the box. Good for simple things imo

**Review by itsalaidbacklife — APPROVED** [review]
Nice work. Please run `npm run version:patch` one time and then push. That will bump the version a patch number and then this is ready to merge

**Comment by sinhashlok** [comment]
> Nice work. Please run `npm run version:patch` one time and then push. That will bump the version a patch number and then this is ready to merge

Patch number updated.

**Review by itsalaidbacklife — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**